content/applications/finance/fiscal_localizations/mexico.rst

```diff
@@ -1053,22 +1021,17 @@ This report can be generated monthly, and a corresponding XML file is created, i
 An additional report is the *Month 13*: a closing balance sheet that shows any adjustments or
```

Comment:
This transition sounds strange to me. I would actually add a heading here:
```suggestion
 Month 13 trial balance
 **********************

The Month 13 report is a closing balance sheet that shows any adjustments or
```
(Not super sure about the wording of the heading itself)
This is unconventional because it would be the only heading in this section BUT I think it would be an easy fix for this clumsy transition